src/DependencyInjection/Configuration.php

```diff
@@ -200,7 +200,7 @@ private function addMailApiProxySection(ArrayNodeDefinition $rootNode): void
                     ->children()
                         ->arrayNode('routing_keys')
                             ->info('Defines the rules that let the MailRequestConsumer listen to routes.')
-                            ->defaultValue(['em_mail_requests_*'])
+                            ->defaultValue(['em_mail_requests.*.*'])
```

Comment:
Same, you can update this `scalarPrototype()->info()` message.

IMO the message info is not correct, what do you think to add the `em_mail_requests` prefix to match `em_mail_requests.{type}.{app_name}` ? 

Can you fix it in this PR ?

Reply:
fixed too